_episodes/04-dependencies.md

```diff
@@ -103,24 +103,24 @@ python countwords.py books/last.txt last.dat
 >
 {: .callout}
 
-The following figure shows the dependencies embodied within our
-Makefile, involved in building the `results.txt` target, after adding
-`countwords.py` and `testzipf.py` as dependencies to their respective target
-files (i.e. how the Makefile should look after completing the rest of the
-exercises in this episode).
+The following figure shows a graph of the dependencies, that are
+involved in building the target `results.txt`. Notice the recently
+added dependencies `countwords.py` and `testzipf.py`.  This is how the
+Makefile should look like after completing the rest of the exercises
+in this episode.
 
 ![results.txt dependencies after adding countwords.py and testzipf.py as dependencies](../fig/04-dependencies.png "results.txt dependencies after adding countwords.py and testzipf.py as dependencies")
 
-> ## Why Don't the `.txt` Files Depend on `countwords.py`?
+> ## Why the `.txt` Files Don't Depend on `countwords.py`?
```

Comment:
This suggestion is grammatically incorrect - we should keep the original heading